src/core.js

```diff
@@ -1,3 +1,10 @@
+// JQuery trim is deprecated, provide a trim method based on String.prototype.trim
+function trim( str ) {
```

Comment:
Please move the function into a local variable and move it down to the functions where it is used

Reply:
Done.